optee-teec/Cargo.lock

```diff
@@ -1,505 +1,505 @@
 # This file is automatically @generated by Cargo.
 # It is not intended for manual editing.
+version = 3
+
```

Comment:
@JesseMK Is this file relevant to the current PR? If it was added by mistake, please reset it, thanks!